.github/workflows/main.yaml

```diff
@@ -51,6 +51,31 @@ jobs:
       - name: Run verify crds test
         run:  make verify
 
+  helm_tests:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Configure Git
+      run: |
+        git config user.name "$GITHUB_ACTOR"
+        git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+    - name: Install Helm
+      uses: azure/setup-helm@v1
+      with:
+        version: v3.6.3
+
+    - name: Run chart-releaser
+      uses: helm/chart-releaser-action@v1.2.1
+      with:
+        charts_dir: charts
+      env:
+        CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
```

Comment:
helm tests should refer to the e2e tests by replacing the installation process of fluent-operator is enough.